tests/cypress/integration/actions_tasks_objects/case_23_canvas_grid_feature.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Canvas grid feature', () => {
+    const caseId = '23';
+    const settingsGridSize = 50;
+    const gridColor = 'Black';
+    const gridOpacity = 49;
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Go to settings.', () => {
+            cy.openSettings();
+        });
+        it('Set "Show grid" to true.', () => {
+            cy.get('.cvat-player-settings-grid').click();
+        });
+        it('Set "Grid size" to 50.', () => {
+            cy.get('.cvat-player-settings-grid-size-input').within(() => {
+                cy.get('[role="spinbutton"]').clear().type(settingsGridSize);
+            });
+        });
+        it('Set "Grid color" to black.', () => {
+            cy.get('.cvat-player-settings-grid-color-input').click();
+            cy.get('.ant-select-dropdown')
+                .not('.ant-select-dropdown-hidden')
+                .contains(new RegExp(`^${gridColor}$`, 'g'))
+                .click();
+        });
+        it('Set "Grid opacity" to 49%.', () => {
+            cy.get('.cvat-player-settings-grid-opacity-input')
+                .click()
+                .within(() => {
+                    cy.get('[role="slider"]').should('have.attr', 'aria-valuenow', gridOpacity);
+                });
+        });
+        it('Canvas has grid drawn, it is black, cells are 50x50px and it has 49% opacity.', () => {
+            cy.get('#cvat_canvas_grid_pattern')
+                .should('exist') // expected <pattern#cvat_canvas_grid_pattern> to exist in the DOM
```

Comment:
What about using 'be.visible' instead, because actually the grid always exists, just has ``display: none`` when disabled

Reply:
Sure. Thanks. 
Id ```cvat_canvas_grid``` has parameter ```display: none```. But other necessary parameters has id ```cvat_canvas_grid_pattern```.
Rewritten.